packages/mobile/src/identity/feelessVerification.ts

```diff
@@ -964,40 +967,47 @@ export function* feelessRequestAttestations(
   } else {
     Logger.debug(
       `${TAG}@feelessRequestAttestations`,
-      `Approving ${numAttestationsRequestsNeeded} new attestations`
+      `Approving and requesting ${numAttestationsRequestsNeeded} new attestations`
     )
 
-    const approveTxResult: Result<CeloTxReceipt, FetchError | TxError> = yield call(
-      [komenciKit, komenciKit.approveAttestations],
-      mtwAddress,
+    const mtwWrapper: MetaTransactionWalletWrapper = yield call(
+      [contractKit.contracts, contractKit.contracts.getMetaTransactionWallet],
+      mtwAddress
+    )
+
+    const approveTx: CeloTransactionObject<boolean> = yield call(
+      [attestationsWrapper, attestationsWrapper.approveAttestationFee],
       numAttestationsRequestsNeeded
     )
 
-    if (!approveTxResult.ok) {
-      Logger.debug(TAG, '@feelessRequestAttestations', 'Failed approve tx')
-      throw approveTxResult.error
-    }
+    const requestTx: CeloTransactionObject<void> = yield call(
+      [attestationsWrapper, attestationsWrapper.request],
+      phoneHash,
+      numAttestationsRequestsNeeded
+    )
 
-    ValoraAnalytics.track(VerificationEvents.verification_request_attestation_approve_tx_sent, {
-      feeless: true,
-    })
+    const approveAndRequestTx = mtwWrapper.executeTransactions([approveTx.txo, requestTx.txo])
```

Comment:
All this can be reverted back to the call to:
```
komenciKit.requestAttestations(...)
```
And the approve step can be removed because the requestAttestations method will take care of crafting the approve transaction as well and pass that to the API.